app/views/datasets/_dataset.html.erb

```diff
@@ -5,26 +5,35 @@
   </td>
   <% if @dashboard %>
     <td class="private">
-      <% if dataset.github_private? %>
+      <% if (dataset.github_private? && dataset.deprecated_resource.eql?(false)) %>
```

Comment:
Could all these repeated `dataset.deprecated_resource.eql?(false)` calls wrap around the entire section in one big if, rather than repeating?